services/apps/premium/members_enrichment_worker/src/activities/enrichment.ts

```
@@ -198,6 +208,186 @@
+      svc.log.info({ memberId }, 'Updating member attributes!')
+      attributes = { ...attributes, ...squashedPayload.attributes }
+      updated = true
+      promises.push(updateMemberAttributes(qx, memberId, attributes))
+    }
+
+    // process reach
+    if (squashedPayload.reach && Object.keys(squashedPayload.reach).length > 0) {
+      svc.log.info({ memberId }, 'Updating member reach!')
+      let reach: IMemberReach
+
+      if (existingMemberData.reach && existingMemberData.reach.total) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)